package.json

```diff
@@ -59,7 +59,6 @@
     "node-forge": "1.0.0",
     "prismjs": "^1.25.0",
     "prism-react-renderer": "1.2.1",
-    "eslint-plugin-react": "7.28.0",
```

Comment:
There's one more in https://github.com/aws-amplify/amplify-ui/blob/625efeb0a205de32c91ccada9e9d986ebb6f2757/canary/react/next/package.json#L17-L19, can you remove that as well?

Reply:
Yup, just realized canary package is not part of the mono repo in terms of workspaces